src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import "./Sidebar.css";
+import { FaSearch } from "react-icons/fa";
+
+import { images } from "../../assets/Images";
+import NavigationMenu from "./components/NavigationMenu";
+
+const Sidebar = () => {
+  const [showUserWrapper, setShowUserWrapper] = useState(false);
+
+  return (
+    <aside className="sidebar">
+      <header className="sidebar-header">
+        <h2 className="logo">
+          Blithe <span>Brains</span>
+        </h2>
+        <button
+          onClick={() => setShowUserWrapper(!showUserWrapper)}
+          class="avatar avatar-sm avatar-round"
+        >
+          <img
+            class="responsive-img"
+            src={images.defaultAvatar}
+            alt="default avatar"
+          />
```

Comment:
UI Suggestion: You could increase the size of the avatar a little bit.

![image](https://user-images.githubusercontent.com/91944618/161799671-b55d3bb4-cad0-4816-ae76-43e78c5ead1d.png)

Reply:
I will try it out